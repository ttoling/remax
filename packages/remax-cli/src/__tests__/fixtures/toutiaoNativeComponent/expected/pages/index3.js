'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var index$1 = require('../index.js');
require('react-reconciler');
require('scheduler');
var React = require('react');
var toutiao = require('../toutiao.js');
var index = require('../index2.js');

var B = index$1.createNativeComponent('b-0');


var _page = function _page() {
  return React.createElement(toutiao.View, null, React.createElement(index.G, null), React.createElement(B, null));
};

var index3 = Page(index$1.createPageConfig(_page), true);

exports.default = index3;
