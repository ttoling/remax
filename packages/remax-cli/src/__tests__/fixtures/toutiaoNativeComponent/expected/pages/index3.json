{
  "usingComponents": {
    "b-0": "../components/b/index"
  }
}